packages/firebase_admob/android/build.gradle

```diff
@@ -30,7 +36,8 @@ android {
         disable 'InvalidPackage'
     }
     dependencies {
-        compile 'com.google.firebase:firebase-core:11.4.+'
-        compile 'com.google.firebase:firebase-ads:11.4.+'
+        // firebase-core is exposed so the google-services plugin will see it.
+        api 'com.google.firebase:firebase-core:11.6.2'
```

Comment:
I'm a bit uneasy about pinning these dependencies to specific versions. Many Flutter apps are going to be using several Firebase-related plugins together, and then they face the challenge of ensuring consistency among transient dependencies like `firebase-core`. Since the other plugins currently (and unfortunately) use `11.4.+`, we have an inconsistency with `11.6.2`. Is upgrading to `11.6.2` necessary? If not, please stay at `11.4.+`. Otherwise, we need to provide consistent versions of the other Firebase plugins as well. Those updates could then bump the Gradle versions as well. And I'd suggest using the version range specifier `[11.6.2,12.0[` to avoid a similar issue in the future, at least until somebody needs v12.

Reply:
Cloud Firestore also needs a bump to `11.6.+`, [see comment](https://github.com/flutter/plugins/pull/282#issuecomment-350100213).

Reply:
<facepalm> Right, this plugin doesn't exist in a vacuum, and is instead part of a suite that needs to work together. I should have accounted for that.

I don't need any recently released features, so there's no need to update the versions in this file now. When they *do* get updated, it should clearly be in concert with the others. Change reverted.